### ISSUE TITLE
RCE via Read Collections Function

### ISSUE THREAD

**Issue by serWazito0** [agentejo/cockpit#1468]
Hi Cockpit Team,

I found that i Can Execute get RCE via php code injection at `read` rules.
This attack can be executed with authentication  and without authentication
## Steps [auth]

1. Go to `dashboard` then Click on `Create Collection` 
<img width="1920" alt="Screen Shot 2021-09-10 at 7 45 50 AM" src="https://user-images.githubusercontent.com/63386763/132805762-7c086ef8-068b-490f-8bd0-6946ac9ab2dc.png">

2. enter any dummy data 
3. at the permission part enable the `read` rule and modify it with the `php` payload.
<img width="1920" alt="Screen Shot 2021-09-10 at 7 48 32 AM" src="https://user-images.githubusercontent.com/63386763/132806071-0216be55-8852-42a4-874b-531e2d55e07b.png">

4. Go to the `Collections`
<img width="1920" alt="Screen Shot 2021-09-10 at 7 50 48 AM" src="https://user-images.githubusercontent.com/63386763/132806189-e597aac6-dfde-40a9-85ee-24183cc9a33f.png">

5. Click On the Created Collection
<img width="1920" alt="Screen Shot 2021-09-10 at 7 52 23 AM" src="https://user-images.githubusercontent.com/63386763/132806307-563eb101-645a-478f-9b06-0da3ed9e8276.png">

6. pass a value to the missing parameter

<img width="1920" alt="Screen Shot 2021-09-10 at 7 53 57 AM" src="https://user-images.githubusercontent.com/63386763/132806455-ddd5b200-bb87-452d-ae7a-c14eebe4606b.png">

Note That the file Created At `storage/collections/rules`
<img width="1056" alt="Screen Shot 2021-09-10 at 7 54 30 AM" src="https://user-images.githubusercontent.com/63386763/132806519-7c68fc23-8302-4275-8f92-e017b9bcf184.png">

[without auth]
You Can Access it without authentication at `/storage/collections/rules/RCE.read.php?cmd=id`
<img width="1920" alt="Screen Shot 2021-09-10 at 7 58 49 AM" src="https://user-images.githubusercontent.com/63386763/132806938-4e84afb6-8242-4556-963d-c7b6820627df.png">

**Comment by aheinze** [comment]
Yes, you can write php code, and this should only be done by admins. You should only give admins the right to create|edit collections.I'll try to find a solution for the direct un-authenticated access

**Comment by serWazito0** [comment]
BTW how can u let the admin execute a PHP code !! if I add this CMS to my website then I gave an admin access to someone he can easily access my server, files, etc.... !!

**Comment by aheinze** [comment]
Why would you give someone admin rights whom you don't trust? he can also delete your account then and take over the cms etc. Just don't give anyone admin rights 🤷‍♂️

**Comment by serWazito0** [comment]
I'm not talking as my side it's an example.
Sometimes The Website owner doesn't change the default Credentials for the CMS, so if the attacker tries it he will log in as admin then execute the PHP code. 🤷‍♂️🤷‍♂️

**Comment by serWazito0** [comment]
Can u just explain to me why u allow the admin to execute PHP code?

**Comment by aheinze** [comment]
This meant for configuring complex content access rules (see comment: https://github.com/agentejo/cockpit/issues/675#issuecomment-367120881)
